internal/errors/http/echo/default.go

```diff
@@ -0,0 +1,69 @@
+// package echo
+//
+// This package implements the default error handler to be
+// used for the api service to match the openapi specification.
+//
+// See: https://echo.labstack.com/docs/error-handling#custom-http-error-handler
+package echo
+
+import (
+	"errors"
+	"fmt"
+	"log/slog"
+	"net/http"
+	"strconv"
+
+	"github.com/labstack/echo/v4"
+	"github.com/openlyinc/pointy"
+	api_public "github.com/podengo-project/idmsvc-backend/internal/api/public"
+	app_context "github.com/podengo-project/idmsvc-backend/internal/infrastructure/context"
+)
+
+func DefaultErrorHandler(err error, c echo.Context) {
+	logger := app_context.LogFromCtx(c.Request().Context())
+	if c.Response().Committed {
+		logger.Warn("response already commited",
+			slog.String("err", err.Error()),
+		)
+		return
+	}
+	errList := make([]api_public.ErrorInfo, 0, 10)
+
+	firstCode := 0
+	response := api_public.ErrorResponse{}
+	for {
+		logger.Debug("encapsulating error", slog.String("msg", err.Error()))
```

Comment:
I find the word "encapsulating" misleading here. "Encapsulating" is bit a synonym to wrapping. But here we are parsing and later unwrapping (almost the opposite). 

So "parsing error" would be more clear to me here. 

Also isn't he message so low level that it would be worth to be "trace" level?